palace.pyx

```diff
@@ -1160,6 +1162,100 @@ cdef class SourceGroup:
         """
         self.impl.destroy()
 
+
+cdef class AuxiliaryEffectSlot:
+    """An effect processor.
+
+    It takes the output mix of zero or more sources, applies DSP for the desired effect
```

Comment:
You should specifies what is "it" before using, in alure#17 specified that is an effect processor.

Reply:
that's 2 lines above.  Could you please proof read the whole thing, run the test (e.g. `tox -e py37`) and give a full review?

Reply:
lemme finish my first commit so I can check his branch without warnings about being overwritten

Reply:
There're problems all over the place.  You should be able to spot them.  Please give a proper review.